src/main/java/org/apache/commons/io/output/DeferredFileOutputStream.java

```diff
@@ -272,4 +274,27 @@ public void writeTo(final OutputStream outputStream) throws IOException {
             }
         }
     }
+
+    /**
+     * Gets the current contents of this byte stream as an {@link InputStream}.
+     * If the data for this output stream has been retained in memory, the
+     * returned stream is backed by buffers of {@code this} stream,
+     * avoiding memory allocation and copy, thus saving space and time.<br>
+     * Otherwise, the returned stream will be one that is created from the data
+     * that has been committed to disk.
+     *
+     * @return the current contents of this output stream.
+     * @see org.apache.commons.io.output.ByteArrayOutputStream.toInputStream()
```

Comment:
@see org.apache.commons.io.output.ByteArrayOutputStream#toInputStream()

Reply:
Ok, I see it now. I needed to run the `javadoc:javadoc` goal. I haven't used maven default goals before, so I was just running `package` and `test` manually. 
It should be fixed now. Sorry for the confusion.